rockcraft.yaml

```diff
@@ -139,11 +144,12 @@ parts:
       mkdir -p ${CRAFT_PART_INSTALL}/usr/lib/ranger
 
       # Unpack trino plugin file
-      tar xvfz target/ranger-2.4.0-trino-plugin.tar.gz \
+      tar xvfz target/ranger-3.0.0-SNAPSHOT-trino-plugin.tar.gz \
         --directory=${CRAFT_PART_INSTALL}/usr/lib/ranger/ \
         --strip-components=1
     stage:
       - usr/lib/ranger
+      - "-usr/lib/ranger/install/conf.templates/enable/ranger-trino-security.xml" # yamllint disable-line
```

Comment:
why does this have this format?

Reply:
It's appended with the `-` to show that it should not be staged. I'm not sure the `"` are strictly necessary but thought it made it more clear than having `--` at the start.

Reply:
Added a comment to make clear.